test/integration/security/user-group-management-mongo.js

```diff
@@ -0,0 +1,900 @@
+module.exports = TestMesh;
+
+function TestMesh() {
+}
+
+TestMesh.prototype.method1 = function ($happn, options, callback) {
+  options.methodName = 'method1';
+  callback(null, options);
+};
+
+if (global.TESTING_USER_MANAGEMENT) return; // When 'requiring' the module above.
```

Comment:
the purpose of this is not so clear to me

Reply:
removed, as none of the tests are calling the exported happner components methods.